cpp/src/io/orc/reader_impl.cu

```diff
@@ -457,6 +460,7 @@ class aggregate_orc_metadata {
           ProtobufReader(sf_data, sf_length)
             .read(per_file_metadata[mapping.source_idx].stripefooters[i]);
           mapping.stripe_info[i].second = &per_file_metadata[mapping.source_idx].stripefooters[i];
+          if (stripe->indexLength == 0) { row_grp_idx_present = false; }
```

Comment:
AFAICT, `row_grp_idx_present` refers to the entire file. Is it possible for `indexLength` to be zero in some (but not all) of the stripes in a file?

Reply:
Yeah, but as we are processing all stripes at once, we can't differentiate between the one has and the one which doesn't. So, even if one of them is not available, we will skip. This scenario mostly arrises when more than one file is being read.